src/pages/my_page/user_profile/MyProjectRegistrations.tsx

```diff
@@ -78,9 +84,9 @@ export const MyProjectRegistrations = ({
           <TablePagination
             rowsPerPageOptions={ROWS_PER_PAGE_OPTIONS}
             component="div"
-            count={projectsQuery.data.size}
+            count={filteredProjects.length}
             rowsPerPage={rowsPerPage}
-            page={page}
+            page={projectsToShow.length <= 0 ? 0 : page}
```

Comment:
Denne kan bli litt skummel siden vi ikke lenger vet at `page` bestemmer hvilken side som er valgt. Vi kan feks øke `page` fra `2` til `3`, men likevel er det side 0 som vises. Om man da endrer filtreringe slik at det blir mange treff kan brukeren plutselig være på side `3` som "default", når det vil være mest naturlig å være på første side. Så tror vi gjør det enklere for oss selv med å si at `page` alltid er valgt side, og hvis det ikke stemmer burde vi sikre at `page` får en gyldig verdi i stedet.

```suggestion
            page={page}
```